server/src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -507,4 +544,114 @@ private static void insertValidationEventListeners(Injector inj) {
         registry.getEventListenerGroup(EventType.PRE_UPDATE).appendListener(listenerProvider.get());
         registry.getEventListenerGroup(EventType.PRE_DELETE).appendListener(listenerProvider.get());
     }
+
+    protected Set<String> typeLabels(Set<String> types) {
+        return types;
+    }
+
+    protected List<String> skus(List<String> skus) {
+        return skus;
+    }
+
+    protected List<String> subscriptionIds(List<String> subscriptionIds) {
+        return subscriptionIds;
+    }
+
+    protected List<String> contracts(List<String> contracts) {
+        return contracts;
+    }
+
+    protected String ownerKey(String key) {
+        return key;
+    }
+
+    protected Product createConsumerWithBindingToMKTProduct(Owner owner) {
+        Consumer c = createConsumer(owner);
+        return createMktProductAndBindItToConsumer(owner, c);
+    }
+
+    protected Product createMktProductAndBindItToConsumer(Owner owner, Consumer c) {
+        String id = String.valueOf(TestUtil.randomInt());
+        Product product = createTransientMarketingProduct(id);
+        product = createProduct(product, owner);
+        Pool pool = createPool(owner, product);
+        createEntitlement(owner, c, pool, null);
+        return product;
+    }
+
+    protected Product createTransientMarketingProduct(String productId) {
```

Comment:
Ok, I will move all utilities back to ConsumerCuratorTest for now, but in case of createTransientMarketingProduct, I can imagine to refactor class ConsumerCuratorSearchTest by using this particular method.